server/helpers/utils.js

```diff
@@ -67,3 +68,51 @@ export const createEllipsis = (text, length = 200) => {
   }
   return `${text.slice(0, length - 3).trim()}...`;
 };
+
+/**
+   * check Validation function
+   * @param {Object} data - data to be validated
+   * @param {Object} rules - rules for validation
+   * @returns {Boolean} true - if validation passes
+   * @returns {Object} error Object - if validation falis
+   */
+export const checkValidation = (data, rules) => {
+  const validation = new Validator(data, rules);
+  if (validation.passes()) {
+    return true;
+  }
+  return {
+    error: {
+      status: 400,
+      message: validation.errors.all()
+    }
+  };
+};
+
+/**
+     * Display error
+     * @param {Object} err
+     * @param {Object} res
+     * @returns {Object} response body - statusCode and errorMessage
+     */
+export const displayError = (err, res) => {
+  const status = 400;
+  res.status(status).json({
+    status,
+    error: err.message,
+  });
+};
+
+/**
+   * validate endpoint
+   * @param {Object} data - data to be validated
+   * @param {Object} rules - rules for validation
+   * @param {Object} res - response body
+   * @param {Object} next - call next function
+   * @returns {Boolean} true - if validation passes
+   * @returns {Object} error Object - if validation fails
+   */
+export const validate = (data, rules, res, next) => {
```

Comment:
Please use proper arguments names to represent the values
next is not a parameter
res is not a parameter